hello/main.go

```diff
@@ -19,25 +24,40 @@ type Response events.APIGatewayProxyResponse
 func Handler(ctx context.Context) (Response, error) {
 	var buf bytes.Buffer
 
-	body, err := json.Marshal(map[string]interface{}{
-		"message": "Go Serverless v1.0! Your function executed successfully!",
+	apiKey := os.Getenv("PD_API_KEY")
+	apiURL := os.Getenv("PD_API_URL")
+	message := "Go Serverless v1.0! Your function executed successfully!"
+
+	// Get sentiment of message
+	form := url.Values{}
+	form.Add("text", message)
+	form.Add("api_key", apiKey)
+	resp, _ := http.Post(
+		apiURL + "/v4/sentiment",
+		"application/x-www-form-urlencoded",
+		strings.NewReader(form.Encode()),
+	)
+	defer resp.Body.Close()
+	body, _ := ioutil.ReadAll(resp.Body)
+
+	responseBody, err := json.Marshal(map[string]interface{}{
+		"message": message,
+		"sentiment": string(body),
```

Comment:
Did you not prefer to keep the communications with the PD API in a different service object ? So far we've been using the hello function as some sort of heartbeat endpoint, maybe it would be better best to keep this code in some nlp package and a function with a clear interface ?

Also, I'm thinking in case we decide to change the NLP engine later, it would maybe be interesting to implement some sort of Adapter pattern. Also we could define a Struct representing a message analysis instead of a plain `map[string]interface{}` if we know how the output is like

so something like

```golang
type NLPAnalysisStruct struct {
  Sentiment string `json:"sentiment"`
  Message string `json:"message"`
}

func AnalyzeMessage(message string) NLPAnalysisStruct {
  adapter.analyzeString()
  ...
  return NLPAnalysisStruct { } 
}
```